app/models/match.rb

```diff
@@ -46,6 +46,8 @@ class Match < ApplicationRecord
   scope :adoptions, -> { where(match_type: :adoption) }
   scope :fosters, -> { where(match_type: :foster) }
 
+  scope :in_foster, -> { fosters.where('? between start_date and end_date', Time.zone.now) }
```

Comment:
I still have some doubts about whether it’s correct to say that if a pet’s match is of the foster type and the current date is between the start date and end date, it is considered in_foster?

Reply:
I think `in_foster` makes sense. 
Are you waiting on feedback on this before adding this to the helper method? I currently note that pets currently in foster do not have a badge showing, yet.

Reply:
@kasugaijin Yes, I just made the adjustment :) 

![image](https://github.com/user-attachments/assets/b816cf26-dc93-456d-938b-10e150c25eeb)